README.md

```diff
@@ -82,6 +82,18 @@ A **Vagrant** configuration is included for development purposes. To use it, com
 - Run `vagrant ssh -c "cd /vagrant && foreman start"`
 - Open `http://mastodon.local` in your browser
 
+To set up **M1/2 MacOS** for native development, complete the following steps:
+
+- Install Ruby 3.2 (using [rbenv](https://github.com/rbenv/rbenv) for easy installation and management of ruby versions)
+- Run `brew install postgresql@15`
```

Comment:
I think you can omit the `15` here and home brew will know to install latest stable?

Reply:
Unfortunately not, at least on my homebrew. I had to explicitly define a version using `@15`.

Reply:
I think 14 is still the version that is used by CI and Docker. I don't think there is anything breaking, but might be good to keep to that version for now

Reply:
Will change it to `14` then - would prefer to keep docs as consistent as possible.

Reply:
@nschonni Updated (or down-dated, I dunno :D )